src/fairseq2/models/wav2vec2/masker.py

```diff
@@ -111,7 +111,7 @@ def forward(
 
         assert temporal_mask is not None
 
-        seqs[temporal_mask] = self.temporal_mask_embed
+        seqs[temporal_mask] = self.temporal_mask_embed.to(dtype=seqs.dtype)
```

Comment:
This has been updated on the main branch a while ago by the way. (see [here](https://github.com/facebookresearch/fairseq2/blob/main/src/fairseq2/models/wav2vec2/masker.py#L114))

Reply:
ah great! Let me rebase this whole PR!